### ISSUE TITLE
Network Overfitting and Convert Model 

### ISSUE THREAD

**Issue by LongLong-Jing** [hx173149/C3D-tensorflow#5]
Hi. Thanks for your amazing work. This code is a great help for me. I trained this network in UCF101 data set. However I got a training accuracy of nearly 98% and test accuracy of 33%. Do you know what caused the overfitting? Also, can you share how you transfer the caffe model to tensorflow mode?  Thanks!

**Comment by hx173149** [comment]
yes if you want to get a high accuracy, you can look at this issue：
https://www.dropbox.com/sh/8wcjrcadx4r31ux/AAAkz3dQ706pPO8ZavrztRCca?dl=0

**Comment by LongLong-Jing** [comment]
Thanks. I will download this model and train again. Also can you give me a hint about how to convert caffe model to tensorflow model? I have struggling about converting model for a long time. Thanks!

**Comment by hx173149** [comment]
@LongLong-Jing  read the caffe model to arrays, and saved it to npy format files, reload these npy files to tensorflow

**Comment by LongLong-Jing** [comment]
@hx173149 Thanks!
